test/controllers/issues_controller_test.rb

```diff
@@ -6,43 +6,51 @@
 class IssuesControllerTest < ActionDispatch::IntegrationTest
   include Devise::Test::IntegrationHelpers
 
+  setup do
```

Comment:
Can we move this to a base controller so we can access and modify these across all controller tests?

Reply:
I don't think all of these are relevant to all controller tests. If we do find another controller than needs this, we can move it into some kind of helper method.